0001-CVE-triage-of-some-libjpeg-CVEs.patch

```diff
@@ -0,0 +1,234 @@
+From a42dc960fcc6de99b2c4926f9a5efed3e5a89be6 Mon Sep 17 00:00:00 2001
+From: CLaire <clairelsouthwell@gmail.com>
+Date: Tue, 5 Jan 2021 11:57:36 -0500
+Subject: [PATCH] CVE triage of some libjpeg CVEs
+
+---
+ active/CVE-2018-11213 | 10 +++++-----
+ active/CVE-2018-11214 | 10 +++++-----
+ active/CVE-2018-11813 | 22 +++++++++++-----------
+ active/CVE-2020-14152 | 22 +++++++++++-----------
+ active/CVE-2020-14153 | 20 ++++++++++----------
+ 5 files changed, 42 insertions(+), 42 deletions(-)
+
+diff --git a/active/CVE-2018-11213 b/active/CVE-2018-11213
+index 5c86ebf841..898cc48375 100644
+--- a/active/CVE-2018-11213
++++ b/active/CVE-2018-11213
+@@ -23,7 +23,7 @@ CVSS:
+  nvd: CVSS:3.0/AV:N/AC:L/PR:N/UI:R/S:U/C:N/I:N/A:H
+ 
+ Patches_libjpeg6b:
+-upstream_libjpeg6b: needs-triage
++upstream_libjpeg6b: deferred
```

Comment:
There are some rare exceptions, but we don't normally use "deferred" for the upstream line. Can you expand on why you chose deferred?